src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/VedtaksperiodeValg.tsx

```diff
@@ -74,6 +76,8 @@ const VedtaksperiodeValg: React.FC<Props> = ({
     const { behandlingErRedigerbar } = useBehandling();
     const { settIkkePersistertKomponent } = useApp();
 
+    const [sanksjonsindex, settSanksjonsindex] = useState<number | undefined>(undefined);
```

Comment:
Tok litt tid før jeg "skjønte" bruken av denne. 

Den gjør jo egentlig to ting; husker index for sanksjon og viser en modal? Ble litt magisk for meg hva den egentlig løste først.

`slettSanksjonsperiode, settVisSlettsanksjonsperiode = useState<{visModal: boolean, index: number|undefined}>({visModal: false})` 

Er dette lettere å lese?  Eller noe "i den duren" ?

Reply:
Ja, veldig bra forslag 👍